docs/DeveloperGuide.md

```diff
@@ -300,32 +300,50 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorEase` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use Case: UC01 - Add student contact**
+
+**MSS**:
+1. Tutor keys in required fields to add student contact
+2. TutorEase adds the student contact
+Use case ends
+
+**Extensions**:
+1a. TutorEase detects bad or wrongly formatted inputs.
+1a1. TutorEase prompts Tutor with correct format
+1a2. Tutor enters new data.
+Steps 1a1 to 1a2 are repeated until the data entered are correct
+Use case resumes from Step 2
```

Comment:
Should we use * to indent the points?

```
* 1a. TutorEase detects bad or wrongly formatted inputs.
    * 1a1. TutorEase prompts Tutor with correct format.
    * 1a2. Tutor enters new data.
    Steps 1a1 to 1a2 are repeated until the data entered are correct.
    Use case resumes from Step 2.
```

It will look something like this.
![image](https://github.com/user-attachments/assets/cc17463a-3271-4086-8b48-4f34e938ce62)

Reply:
I think this would be better 👍